.github/actions/issue-validator/src/index.js

```diff
@@ -17,17 +18,22 @@ async function run() {
 
     const { body, labels, number: issueNumber } = issue
 
+    const isBugReport = labels.some((label) => label.name === bugReportLabel)
+
     const isManuallyLabeled = labels.some(
       (label) => label.name === verifyCanaryLabel
     )
 
-    const isBugReport = labels.some((label) => label.name === bugReportLabel)
+    if (!isBugReport || !isManuallyLabeled) {
```

Comment:
Should this `||` be `&&`?

Reply:
No, basically it allows us to ignore either non-bug reports (issues with the "template: bug" label) OR old issues we manually added the "please verify canary" label. Those old issues might not have "template: bug" yet.

Reply:
Yes, it should be `&&`... :sweat_smile: